man/docker-cp.1.md

```diff
@@ -147,6 +148,17 @@ to a directory `/etc/my-app.d` in a container:
 The above command will copy the contents of the local `/config` directory into
 the directory `/etc/my-app.d` in the container.
 
+Third, if you want to copy a symbol link into a container, what you want is to
```

Comment:
@WeiZhang555 Wow! Thank you for doing that. Really cool.  

I missed this example in my review, it has some spelling and construction issues. Try this:

```markdown
Finally, if you want to copy a symbolic link into a container, you typically
want to  copy the linked target and not the link itself. To copy the target, use
the `-L` option, for example:

		$ ln -s /tmp/somefile /tmp/somefile.ln
		$ docker cp -L /tmp/somefile.ln myappcontainer:/tmp/

This command copies content of the local `/tmp/somefile` into the file
`/tmp/somefile.ln` in the container. Without `-L` option, the `/tmp/somefile.ln`
preserves its symbolic link but not its content.
```